dashboard/src/app/sitemap.ts

```diff
@@ -0,0 +1,102 @@
+import { MetadataRoute } from 'next';
+import { env } from '@/lib/env';
+
+export default function sitemap(): MetadataRoute.Sitemap {
+  const baseUrl = env.NEXT_PUBLIC_BASE_URL;
+
+  return [
+    /****************** PUBLIC PAGES ******************/
+
+    {
+      url: baseUrl,
+      lastModified: new Date(),
```

Comment:
Is it correct/good practice to let `lastModified` be set to `new Date()` on every page?